Fails to build on `npm run build`

npm run build fails with the following output
<details>

```
node_modules/@apollo/federation/dist/composition/DirectiveMetadata.d.ts:1:221 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { ASTNode, DirectiveNode, GraphQLSchema, InterfaceTypeDefinitionNode, InterfaceTypeExtensionNode, ObjectTypeDefinitionNode, ObjectTypeExtensionNode, UnionTypeDefinitionNode, UnionTypeExtensionNode, VisitFn } from 'graphql';
                                                                                                                                                                                                                              ~~~~~~~~~

node_modules/@apollo/federation/dist/composition/compose.d.ts:1:109 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, GraphQLError, TypeDefinitionNode, DirectiveDefinitionNode, TypeExtensionNode } from 'graphql';
                                                                                                              ~~~~~~~~~

node_modules/@apollo/federation/dist/composition/normalize.d.ts:1:30 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode } from 'graphql';
                               ~~~~~~~~~

node_modules/@apollo/federation/dist/composition/rules.d.ts:1:47 - error TS2307: Cannot find module 'graphql/validation/ValidationContext' or its corresponding type declarations.

1 export declare const compositionRules: import("graphql/validation/ValidationContext").SDLValidationRule[];
                                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@apollo/federation/dist/composition/types.d.ts:1:122 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode, FieldDefinitionNode, DirectiveDefinitionNode, DirectiveNode, FieldNode, InlineFragmentNode } from 'graphql';
                                                                                                                           ~~~~~~~~~

node_modules/@apollo/federation/dist/composition/utils.d.ts:1:294 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { StringValueNode, NameNode, DocumentNode, DirectiveNode, GraphQLNamedType, GraphQLError, GraphQLSchema, GraphQLObjectType, GraphQLField, TypeDefinitionNode, TypeExtensionNode, ASTNode, DirectiveDefinitionNode, GraphQLDirective, OperationTypeNode, NonNullTypeNode, NamedTypeNode } from 'graphql';
                                                                                                                                                                                                                                                                                                       ~~~~~~~~~

node_modules/@apollo/federation/dist/composition/utils.d.ts:14:144 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

14 export declare function findSelectionSetOnNode(node: Maybe<ASTNodeWithDirectives>, directiveName: string, printedSelectionSet: string): import("graphql").ValueNode | undefined;
                                                                                                                                                  ~~~~~~~~~

node_modules/@apollo/federation/dist/composition/utils.d.ts:59:132 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

59 export declare function findTypeNodeInServiceList(typeName: string, serviceName: string, serviceList: ServiceDefinition[]): import("graphql").DefinitionNode | undefined;
                                                                                                                                      ~~~~~~~~~

node_modules/@apollo/gateway/dist/config.d.ts:1:45 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLError, GraphQLSchema } from "graphql";
                                              ~~~~~~~~~

node_modules/@apollo/gateway/dist/datasources/LocalGraphQLDataSource.d.ts:2:45 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

2 import { GraphQLSchema, DocumentNode } from 'graphql';
                                              ~~~~~~~~~

node_modules/@apollo/gateway/dist/executeQueryPlan.d.ts:2:38 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

2 import { GraphQLFieldResolver } from 'graphql';
                                       ~~~~~~~~~

node_modules/@apollo/gateway/dist/index.d.ts:4:31 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

4 import { GraphQLSchema } from 'graphql';
                                ~~~~~~~~~

node_modules/@apollo/gateway/dist/operationContext.d.ts:1:94 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode, FragmentDefinitionNode, GraphQLSchema, OperationDefinitionNode } from 'graphql';
                                                                                               ~~~~~~~~~

node_modules/@apollo/query-planner/dist/FieldSet.d.ts:1:81 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { FieldNode, GraphQLCompositeType, GraphQLField, SelectionSetNode } from 'graphql';
                                                                                  ~~~~~~~~~

node_modules/@apollo/query-planner/dist/QueryPlan.d.ts:1:57 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { SelectionNode as GraphQLJSSelectionNode } from 'graphql';
                                                          ~~~~~~~~~

node_modules/@apollo/query-planner/dist/QueryPlanningContext.d.ts:1:209 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { FieldNode, FragmentDefinitionNode, GraphQLAbstractType, GraphQLCompositeType, GraphQLField, GraphQLObjectType, GraphQLSchema, OperationDefinitionNode, SelectionSetNode, VariableDefinitionNode } from "graphql";
                                                                                                                                                                                                                  ~~~~~~~~~

node_modules/@apollo/query-planner/dist/Scope.d.ts:1:72 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DirectiveNode, GraphQLCompositeType, GraphQLObjectType } from "graphql";
                                                                         ~~~~~~~~~

node_modules/@apollo/query-planner/dist/buildQueryPlan.d.ts:1:116 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode, FragmentDefinitionNode, GraphQLCompositeType, GraphQLSchema, OperationDefinitionNode } from 'graphql';
                                                                                                                     ~~~~~~~~~

node_modules/@apollo/query-planner/dist/composedSchema/buildComposedSchema.d.ts:1:45 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode, GraphQLSchema } from 'graphql';
                                              ~~~~~~~~~

node_modules/@apollo/query-planner/dist/composedSchema/metadata.d.ts:1:80 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { FieldNode, InlineFragmentNode, GraphQLField, GraphQLObjectType } from 'graphql';
                                                                                 ~~~~~~~~~

node_modules/@apollo/query-planner/dist/composedSchema/toAPISchema.d.ts:1:31 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema } from 'graphql';
                                ~~~~~~~~~

node_modules/@apollo/query-planner/dist/index.d.ts:7:31 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

7 import { GraphQLSchema } from 'graphql';
                                ~~~~~~~~~

node_modules/@apollo/query-planner/dist/snapshotSerializers/astSerializer.d.ts:1:25 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { ASTNode } from 'graphql';
                          ~~~~~~~~~

node_modules/@apollo/subgraph/dist/buildSubgraphSchema.d.ts:1:45 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode, GraphQLSchema } from 'graphql';
                                              ~~~~~~~~~

node_modules/@apollo/subgraph/dist/directives.d.ts:1:270 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLDirective, GraphQLNamedType, GraphQLInputObjectType, DirectiveNode, GraphQLField, FieldDefinitionNode, InputValueDefinitionNode, SchemaDefinitionNode, TypeSystemExtensionNode, TypeDefinitionNode, ExecutableDefinitionNode, DirectiveDefinitionNode } from 'graphql';
                                                                                                                                                                                                                                                                               ~~~~~~~~~

node_modules/@apollo/subgraph/dist/printSubgraphSchema.d.ts:1:49 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, GraphQLNamedType } from 'graphql';
                                                  ~~~~~~~~~

node_modules/@apollographql/apollo-tools/lib/buildServiceDefinition.d.ts:1:59 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, DocumentNode, GraphQLError } from "graphql";
                                                            ~~~~~~~~~

node_modules/@apollographql/apollo-tools/lib/schema/resolveObject.d.ts:1:47 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLResolveInfo, FieldNode } from "graphql";
                                                ~~~~~~~~~

node_modules/@apollographql/apollo-tools/lib/schema/resolverMap.d.ts:1:38 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLFieldResolver } from "graphql";
                                       ~~~~~~~~~

node_modules/@apollographql/apollo-tools/lib/utilities/graphql.d.ts:1:78 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { ASTNode, TypeDefinitionNode, TypeExtensionNode, DocumentNode } from "graphql";
                                                                               ~~~~~~~~~

node_modules/@graphql-tools/mock/MockStore.d.ts:1:31 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema } from 'graphql';
                                ~~~~~~~~~

node_modules/@graphql-tools/mock/addMocksToSchema.d.ts:1:31 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema } from 'graphql';
                                ~~~~~~~~~

node_modules/@graphql-tools/mock/types.d.ts:1:48 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { ExecutionResult, GraphQLSchema } from 'graphql';
                                                 ~~~~~~~~~

node_modules/@graphql-tools/utils/Interfaces.d.ts:1:1022 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, GraphQLField, GraphQLInputType, GraphQLNamedType, GraphQLResolveInfo, GraphQLScalarType, DocumentNode, FieldNode, GraphQLEnumValue, GraphQLEnumType, GraphQLUnionType, GraphQLArgument, GraphQLInputField, GraphQLInputObjectType, GraphQLInterfaceType, GraphQLObjectType, GraphQLDirective, FragmentDefinitionNode, SelectionNode, ExecutionResult as GraphQLExecutionResult, GraphQLOutputType, FieldDefinitionNode, GraphQLFieldConfig, GraphQLInputFieldConfig, GraphQLArgumentConfig, GraphQLEnumValueConfig, GraphQLScalarSerializer, GraphQLScalarValueParser, GraphQLScalarLiteralParser, ScalarTypeDefinitionNode, ScalarTypeExtensionNode, EnumTypeDefinitionNode, EnumTypeExtensionNode, GraphQLIsTypeOfFn, ObjectTypeDefinitionNode, ObjectTypeExtensionNode, InterfaceTypeExtensionNode, InterfaceTypeDefinitionNode, GraphQLTypeResolver, UnionTypeDefinitionNode, UnionTypeExtensionNode, InputObjectTypeExtensionNode, InputObjectTypeDefinitionNode, GraphQLType, Source, DefinitionNode, OperationTypeNode } from 'graphql';
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               ~~~~~~~~~

node_modules/@graphql-tools/utils/addTypes.d.ts:1:67 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, GraphQLNamedType, GraphQLDirective } from 'graphql';
                                                                    ~~~~~~~~~

node_modules/@graphql-tools/utils/astFromValueUntyped.d.ts:1:27 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { ValueNode } from 'graphql';
                            ~~~~~~~~~

node_modules/@graphql-tools/utils/build-operation-for-field.d.ts:1:75 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, OperationDefinitionNode, OperationTypeNode } from 'graphql';
                                                                            ~~~~~~~~~

node_modules/@graphql-tools/utils/collectFields.d.ts:1:103 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, FragmentDefinitionNode, GraphQLObjectType, SelectionSetNode, FieldNode } from 'graphql';
                                                                                                        ~~~~~~~~~

node_modules/@graphql-tools/utils/comments.d.ts:1:78 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { StringValueNode, ASTNode, NameNode, DefinitionNode, Location } from 'graphql';
                                                                               ~~~~~~~~~

node_modules/@graphql-tools/utils/errors.d.ts:1:30 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLError } from 'graphql';
                               ~~~~~~~~~

node_modules/@graphql-tools/utils/fields.d.ts:1:74 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLFieldConfigMap, GraphQLFieldConfig, GraphQLSchema } from 'graphql';
                                                                           ~~~~~~~~~

node_modules/@graphql-tools/utils/filterSchema.d.ts:1:31 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema } from 'graphql';
                                ~~~~~~~~~

node_modules/@graphql-tools/utils/fixSchemaAst.d.ts:1:51 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, BuildSchemaOptions } from 'graphql';
                                                    ~~~~~~~~~

node_modules/@graphql-tools/utils/forEachDefaultValue.d.ts:1:31 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema } from 'graphql';
                                ~~~~~~~~~

node_modules/@graphql-tools/utils/forEachField.d.ts:1:31 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema } from 'graphql';
                                ~~~~~~~~~

node_modules/@graphql-tools/utils/get-directives.d.ts:1:345 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, GraphQLNamedType, GraphQLField, GraphQLInputField, GraphQLFieldConfig, GraphQLInputFieldConfig, GraphQLSchemaConfig, GraphQLObjectTypeConfig, GraphQLInterfaceTypeConfig, GraphQLUnionTypeConfig, GraphQLScalarTypeConfig, GraphQLEnumTypeConfig, GraphQLInputObjectTypeConfig, GraphQLEnumValue, GraphQLEnumValueConfig } from 'graphql';
                                                                                                                                                                                                                                                                                                                                                          ~~~~~~~~~

node_modules/@graphql-tools/utils/get-fields-with-directives.d.ts:1:30 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode } from 'graphql';
                               ~~~~~~~~~

node_modules/@graphql-tools/utils/get-implementing-types.d.ts:1:31 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema } from 'graphql';
                                ~~~~~~~~~

node_modules/@graphql-tools/utils/getArgumentValues.d.ts:1:74 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLField, GraphQLDirective, DirectiveNode, FieldNode } from 'graphql';
                                                                           ~~~~~~~~~

node_modules/@graphql-tools/utils/getResolversFromSchema.d.ts:1:31 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema } from 'graphql';
                                ~~~~~~~~~

node_modules/@graphql-tools/utils/getResponseKeyFromInfo.d.ts:1:36 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLResolveInfo } from 'graphql';
                                     ~~~~~~~~~

node_modules/@graphql-tools/utils/heal.d.ts:1:67 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLDirective, GraphQLNamedType, GraphQLSchema } from 'graphql';
                                                                    ~~~~~~~~~

node_modules/@graphql-tools/utils/helpers.d.ts:1:25 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { ASTNode } from 'graphql';
                          ~~~~~~~~~

node_modules/@graphql-tools/utils/implementsAbstractType.d.ts:1:44 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLType, GraphQLSchema } from 'graphql';
                                             ~~~~~~~~~

node_modules/@graphql-tools/utils/isDocumentNode.d.ts:1:30 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode } from 'graphql';
                               ~~~~~~~~~

node_modules/@graphql-tools/utils/loaders.d.ts:1:65 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode, GraphQLSchema, BuildSchemaOptions } from 'graphql';
                                                                  ~~~~~~~~~

node_modules/@graphql-tools/utils/mapSchema.d.ts:1:113 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLObjectType, GraphQLSchema, GraphQLInputObjectType, GraphQLInterfaceType, GraphQLEnumType } from 'graphql';
                                                                                                                  ~~~~~~~~~

node_modules/@graphql-tools/utils/parse-graphql-json.d.ts:1:30 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { ParseOptions } from 'graphql';
                               ~~~~~~~~~

node_modules/@graphql-tools/utils/parse-graphql-sdl.d.ts:1:56 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode, ASTNode, StringValueNode } from 'graphql';
                                                         ~~~~~~~~~

node_modules/@graphql-tools/utils/print-schema-with-directives.d.ts:1:584 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, GraphQLNamedType, DirectiveNode, FieldDefinitionNode, InputValueDefinitionNode, GraphQLArgument, EnumValueDefinitionNode, GraphQLDirective, DirectiveDefinitionNode, SchemaDefinitionNode, SchemaExtensionNode, GraphQLObjectType, ObjectTypeDefinitionNode, GraphQLField, GraphQLInterfaceType, InterfaceTypeDefinitionNode, UnionTypeDefinitionNode, GraphQLUnionType, GraphQLInputObjectType, InputObjectTypeDefinitionNode, GraphQLInputField, GraphQLEnumType, GraphQLEnumValue, EnumTypeDefinitionNode, GraphQLScalarType, ScalarTypeDefinitionNode, DocumentNode } from 'graphql';
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         ~~~~~~~~~

node_modules/@graphql-tools/utils/prune.d.ts:1:31 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema } from 'graphql';
                                ~~~~~~~~~

node_modules/@graphql-tools/utils/renameType.d.ts:1:153 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLEnumType, GraphQLInputObjectType, GraphQLInterfaceType, GraphQLObjectType, GraphQLNamedType, GraphQLScalarType, GraphQLUnionType } from 'graphql';
                                                                                                                                                          ~~~~~~~~~

node_modules/@graphql-tools/utils/rewire.d.ts:1:52 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLDirective, GraphQLNamedType } from 'graphql';
                                                     ~~~~~~~~~

node_modules/@graphql-tools/utils/rootTypes.d.ts:1:69 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLObjectType, GraphQLSchema, OperationTypeNode } from 'graphql';
                                                                      ~~~~~~~~~

node_modules/@graphql-tools/utils/selectionSets.d.ts:1:34 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { SelectionSetNode } from 'graphql';
                                   ~~~~~~~~~

node_modules/@graphql-tools/utils/stub.d.ts:1:159 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLObjectType, GraphQLInterfaceType, GraphQLInputObjectType, GraphQLNamedType, TypeNode, GraphQLType, GraphQLOutputType, GraphQLInputType } from 'graphql';
                                                                                                                                                                ~~~~~~~~~

node_modules/@graphql-tools/utils/transformInputValue.d.ts:1:34 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLInputType } from 'graphql';
                                   ~~~~~~~~~

node_modules/@graphql-tools/utils/types.d.ts:1:101 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLEnumType, GraphQLInputObjectType, GraphQLNamedType, GraphQLScalarType, visit } from 'graphql';
                                                                                                      ~~~~~~~~~

node_modules/@graphql-tools/utils/updateArgument.d.ts:1:72 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLInputType, ArgumentNode, VariableDefinitionNode } from 'graphql';
                                                                         ~~~~~~~~~

node_modules/@graphql-tools/utils/validate-documents.d.ts:1:76 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, GraphQLError, ValidationContext, ASTVisitor } from 'graphql';
                                                                             ~~~~~~~~~

node_modules/@graphql-tools/utils/validate-documents.d.ts:10:54 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

10 export declare function createDefaultRules(): import("graphql").ValidationRule[];
                                                        ~~~~~~~~~

node_modules/@graphql-tools/utils/visitResult.d.ts:1:45 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, GraphQLError } from 'graphql';
                                              ~~~~~~~~~

node_modules/apollo-graphql/lib/operationId.d.ts:1:30 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode } from "graphql";
                               ~~~~~~~~~

node_modules/apollo-graphql/lib/schema/GraphQLSchemaValidationError.d.ts:1:30 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLError } from "graphql";
                               ~~~~~~~~~

node_modules/apollo-graphql/lib/schema/buildSchemaFromSDL.d.ts:1:45 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { DocumentNode, GraphQLSchema } from "graphql";
                                              ~~~~~~~~~

node_modules/apollo-graphql/lib/schema/resolveObject.d.ts:1:47 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLResolveInfo, FieldNode } from "graphql";
                                                ~~~~~~~~~

node_modules/apollo-graphql/lib/schema/resolverMap.d.ts:1:57 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLFieldResolver, GraphQLScalarType } from "graphql";
                                                          ~~~~~~~~~

node_modules/apollo-graphql/lib/schema/transformSchema.d.ts:1:49 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, GraphQLNamedType } from "graphql";
                                                  ~~~~~~~~~

node_modules/apollo-server-core/dist/ApolloServer.d.ts:1:45 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, DocumentNode } from 'graphql';
                                              ~~~~~~~~~

node_modules/apollo-server-core/dist/gql.d.ts:1:35 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import type { DocumentNode } from 'graphql';
                                    ~~~~~~~~~

node_modules/apollo-server-core/dist/graphqlOptions.d.ts:1:142 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import type { GraphQLSchema, ValidationContext, GraphQLFieldResolver, DocumentNode, GraphQLError, GraphQLFormattedError, ParseOptions } from 'graphql';
                                                                                                                                               ~~~~~~~~~

node_modules/apollo-server-core/dist/plugin/usageReporting/options.d.ts:1:49 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import type { GraphQLError, DocumentNode } from 'graphql';
                                                  ~~~~~~~~~

node_modules/apollo-server-core/dist/requestPipeline.d.ts:1:118 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { GraphQLSchema, GraphQLFieldResolver, DocumentNode, GraphQLError, GraphQLFormattedError, ParseOptions } from 'graphql';
                                                                                                                       ~~~~~~~~~

node_modules/apollo-server-core/dist/types.d.ts:1:64 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import type { GraphQLSchema, DocumentNode, ParseOptions } from 'graphql';
                                                                 ~~~~~~~~~

node_modules/apollo-server-errors/dist/index.d.ts:1:86 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

1 import { ASTNode, GraphQLError, GraphQLFormattedError, Source, SourceLocation } from 'graphql';
                                                                                       ~~~~~~~~~

node_modules/apollo-server-types/dist/index.d.ts:2:185 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

2 import type { GraphQLSchema, ValidationContext, ASTVisitor, GraphQLFormattedError, OperationDefinitionNode, DocumentNode, GraphQLError, GraphQLResolveInfo, GraphQLCompositeType } from 'graphql';
                                                                                                                                                                                          ~~~~~~~~~


Found 86 errors.
```

</details>

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -12,6 +12,7 @@
     "noImplicitThis": true,
     "alwaysStrict": true,
     "noUnusedLocals": false,
+    "skipLibCheck": true,
     "noUnusedParameters": false,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": false,

